assets/js/ModuleCategory.js

```diff
@@ -47,15 +52,13 @@ export default class ModuleCategory extends React.Component {
 
     var sortIcon = isOpen ? 'icon-sort-up-vl-gray' : 'icon-sort-down-vl-gray';
 
-    // ??? are these the correct icons per category ???
     const icons = {
       'Add data': 'database',
-      'Analyze': 'notepad',
+      'Analyse': 'notepad',         // Inconsistency with 'Analyze' elsewhere
```

Comment:
Didn't we get rid of 'Analyse' category?

Reply:
I was not aware of that, some modules still load into that category, however.

Should I search & replace any instances of "Analyse" with "Analyse"?